vshard/storage/init.lua

```diff
@@ -232,7 +232,10 @@ if not M then
         -- Condition variable fired each time a bucket locked for
         -- RW refs reaches 0 of the latter.
         bucket_rw_lock_is_ready_cond = lfiber.cond(),
-
+        -- This table contains true/false values for each replicaset depending
+        -- on whether all of its buckets have BACTIVE status during rebalancing
+        -- stage.
+        replicasets_active_completeness = {},
```

Comment:
I propose to fight with deduplication of the logs in the scope of https://github.com/tarantool/vshard/issues/588, not here. I don't see much sense in such sophisticated deduplication, if the prev error is the same as the current one, then we don't print it, that's seems enough

The pairs over the same table of `replicasets` will work in the same order on every iteration anway (please, correct me, if that's not right). So, you'll encounter problems on the same replicaset every iteration of the `rebalancer`, right?